modules/tinymce/src/plugins/lists/test/ts/browser/ContentEditableFalseActionsTest.ts

```diff
@@ -0,0 +1,102 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinySelections, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/lists/Plugin';
+
+describe('browser.tinymce.plugins.lists.ContentEditableFalseActionsTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'lists',
+    toolbar: 'numlist bullist',
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Plugin ], true);
+
+  it('TINY-8920: List toolbar buttons are disabled when in noneditable OL list', () => {
+    const editor = hook.editor();
+    const content = '<ol contenteditable="false">\n' +
+      '<li contenteditable="true">editable</li>\n' +
+      '<li>noneditable</li>\n' +
+      '<li contenteditable="true">editable</li>\n' +
+    '</ol>';
+    editor.setContent(content);
+    TinySelections.setCursor(editor, [ 1, 2, 0 ], 0);
+    TinyUiActions.clickOnToolbar(editor, 'button[aria-label="Numbered list"]');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+    TinyUiActions.clickOnToolbar(editor, 'button[aria-label="Bullet list"]');
```

Comment:
Checking numbered list vs bullet list should be 2 separate tests, as one could fail and then it's going to be hard to diagnose which one failed.

Reply:
Since they both use the same `NodeChangeEvent` to disable them in `contenteditable="false"` contexts, if one fails (by changing the content) they would both fail.

Reply:
That's not always going to be the case though, someone in the next PR could change that and then the tests wouldn't cover that...